custom/abt/reports/data_sources/sms_case.json

```diff
@@ -0,0 +1,1010 @@
+                "Inala": 3506,
+                "Kikubiji": 6309,
+                "Malya": 4307,
+                "Mwamashimba": 6997,
+                "Ngumo": 7456,
+                "Nyamikoma": 5888,
+                "Nyamilama": 6529,
+                "Micheweni": 3192,
+                "Ngejani": 5260,
+                "Uzi": 888,
+                "Mahonda": 5736,
+                "Kibuteni": 4073,
+                "Mombasa": 4457,
+                "Selemu": 2757,
+                "Shakani": 3166,
+                "Ndijani Mseweni": 5260
+              },
+              "type": "switch",
+              "switch_on": {
+                "value_expression": {
+                  "type": "property_path",
+                  "property_path": [
+                    "user_data",
+                    "level_3_name"
+                  ]
+                },
+                "type": "related_doc",
+                "related_doc_type": "CommCareUser",
+                "doc_id_expression": {
+                  "type": "property_name",
+                  "property_name": "owner_id"
+                }
+              }
+            }
+          },
+          "type": "switch",
+          "switch_on": {
+            "value_expression": {
+              "type": "property_path",
+              "property_path": [
+                "user_data",
+                "country"
+              ]
+            },
+            "type": "related_doc",
+            "related_doc_type": "CommCareUser",
+            "doc_id_expression": {
+              "type": "property_name",
+              "property_name": "owner_id"
+            }
+          }
+        },
+        "column_id": "level_3_target_number"
+      },
+      {
+        "display_name": "country_target_number",
+        "datatype": "integer",
+        "type": "expression",
+        "is_primary_key": false,
+        "transform": {},
+        "is_nullable": true,
+        "expression": {
+          "default": 0,
+          "cases": {
+            "Senegal": 31816,
+            "Rwanda": 144417,
+            "Zambia": 539348,
+            "Ethiopia": 670303,
+            "Mali": 135717,
+            "Madagascar": 72120,
+            "Tanzania": 502934
+          },
+          "type": "switch",
+          "switch_on": {
+            "value_expression": {
+              "type": "property_path",
+              "property_path": [
+                "user_data",
+                "country"
+              ]
+            },
+            "type": "related_doc",
+            "related_doc_type": "CommCareUser",
+            "doc_id_expression": {
+              "type": "property_name",
+              "property_name": "owner_id"
+            }
+          }
+        },
+        "column_id": "country_target_number"
+      }
+    ],
+    "description": "",
+    "display_name": "SMS Indicators (static)",
+    "named_filters": {},
+    "referenced_doc_type": "XFormInstance",
+    "table_id": "static-sms-indicators-001"
```

Comment:
One little thing to note: If this data source is ever in the same domain as the `sms.json` data source, there will be unpredictable behavior, because their `table_id`s are the same. But, keeping the ids the same allows us to reuse the static reports without duplicating them. So I think I'm in favor of leaving this table_id as is, but something to keep in mind if we add this to additional domains.